monitoring/configure-metrics-tools.sh

```diff
@@ -49,6 +49,11 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
 # set core context
 kubectl config use-context $ctx
 
+: --------------------------------------------------------------------
+: Configure APF for prometheus traffic to the WDS API server
+: --------------------------------------------------------------------
+kubectl --context $wds apply -f ${SCRIPT_DIR}/configuration/prometheus-pod-exempt.yaml
```

Comment:
Don't you want to this FlowSchema to go in the KubeFlex hosting cluster? Isn't that where the Prometheus pod runs?